### DISCUSSION TITLE
Northlight Tool

### DISCUSSION THREAD

**Discussion by Vamp1r3-42** [TomEvin/neat#5] [General]
Is the norhtlight tool still available somewhere for download?

**Comment by YoungSaiyan** [comment]
I was digging for it here too and finally found in this reddit post. 
https://www.reddit.com/r/controlgame/comments/17t8yni/control_model_export_to_blender/

The user posted a google drive link with the tool and few other goodies.
Google Drive link: https://drive.google.com/drive/folders/15E_Wfvv3tYOldbYUvwTVMiXISmHLdaK1?usp=drive_link

**Reply by ValFan771** [reply]
Absolute legend, thank you for the digging!
